packages/@lwc/ssr-runtime/src/lightning-element.ts

```diff
@@ -29,31 +29,33 @@ interface PropsAvailableAtConstruction {
     tagName: string;
 }
 
+export const SYMBOL__SET_INTERNALS = Symbol();
+
 export class LightningElement implements PropsAvailableAtConstruction {
     static renderMode?: 'light' | 'shadow';
 
     isConnected = false;
     className = '';
-    // TODO [W-14977927]: protect internals from userland
-    private __attrs!: Attributes;
-    private __classList: ClassList | null = null;
+
     // Using ! because it's assigned in the constructor via `Object.assign`, which TS can't detect
     tagName!: string;
 
+    #__attrs!: Attributes;
+    #__classList: ClassList | null = null;
```

Comment:
```suggestion
    #attrs!: Attributes;
    #classList: ClassList | null = null;
```
If we're using private class fields, we don't need the "fake private" prefix in the names.